flang/documentation/BurnsideToFIR.md

```diff
@@ -428,7 +428,7 @@ Fortran into FIR in a semantics preserving way.
     // create a temporary to hold E(V)
     %v = ... : !fir.array<4:i32>
     %8 = fir.alloca !fir.array<4:f32> : !fir.ref<!fir.array<4:f32>>
-    fir.loop %i = %c1 to %c4 unordered {
+    fir.doloop %i = %c1 to %c4 unordered {
```

Comment:
Should be `do_loop` in the syntactic form like this. Although the `_` is omitted in the camelcase class name.